package.json

```diff
@@ -23,7 +23,8 @@
   "scripts": {
     "test": "jest --coverage",
     "check": "yarn lint && yarn test && yarn build",
-    "build": "yarn tsc --noEmit && yarn rollup -c",
+    "build": "yarn tsc --noEmit && yarn rollup -c && yarn build:cdn:version",
+    "build:cdn:version": "jq -r .version package.json > cdn/.version",
```

Comment:
This could be done also with `echo ${npm_package_version} > cdn/.version`, if you do want to avoid `jq` as a build dependency.